srml/contracts/src/tests.rs

```diff
@@ -35,7 +35,7 @@ use sr_primitives::{
 };
 use support::{
 	assert_ok, assert_err, impl_outer_dispatch, impl_outer_event, impl_outer_origin, parameter_types,
-	storage::child, StorageMap, StorageValue, traits::{Currency, Get},
+	storage::child, StorageMap, StorageValue, traits::{Currency, Get, ExistenceRequirement},
```

Comment:
I would _maybe_ derive `Default` for `ExistenceRequirement ` and use it here + it makes sense as imo it _actually is the default_ based on the comment above.